<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="stockList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>601</width>
     <height>241</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>종목명</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="text">
       <string>시작</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopBtn">
      <property name="text">
       <string>중지</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QComboBox" name="strategyList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>알고리즘</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="showProcess">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
